applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -581,6 +588,13 @@ For example:
 
       west build -b nrf5340dk/nrf5340/cpuapp -p -- -Dmatter_bridge_SHIELD=nrf7002ek -DSB_CONFIG_WIFI_PATCHES_EXT_FLASH_STORE=y -DSB_CONFIG_DFU_MULTI_IMAGE_PACKAGE_WIFI_FW_PATCH=y -DSB_CONFIG_WIFI_NRF700X=y -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3
 
+
+You can program nRF54h20 board with nRf7002 EB,using following command:
```

Comment:
Could you expand the description a little bit to look similar (or the same), as here: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/matter/lock/README.html#building_and_running. I mean this part: "To use nrf54H20 DK with the nrf7002ek shield attached  ..."